spec/document_builder/getting_started_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+describe 'document_builder_getting_started' do
+  before :all do
+    @test_manager = TestingApiOnlyfficeCom::TestManager.new(suite_name: 'Document Builder Getting Started', plan_name: @config.to_s)
+  end
+
+  before :each do
+    @instance = TestingApiOnlyfficeCom::TestInstance.new(@config)
+    @api_page = @instance.go_to_main_page
+  end
+
+  describe 'download_libraries' do
+    before :each do
+      @introduction_page = @api_page.go_to_document_builder_introduction
```

Comment:
@introduction_page is used only in `:each`, so `introduction_page` without `@` is enought